schemas/package-v1alpha1.schema.json

```diff
@@ -954,7 +954,7 @@
       ],
       "title": "ProtocolMapper"
     },
-    "Status": {
+    "StatusObject": {
```

Comment:
A note for reviewers: this conversation happened automatically by kubernetes-fluent-client. This is due to duplicated `status` name - one of which is an Object and one is String.